docs/100.yaml

```diff
@@ -385,7 +385,7 @@
 93:
   title  : The Birthday Problem
   links  :
-    mathlib archive: https://github.com/leanprover-community/mathlib4/blob/master/Archive/Wiedijk100Theorems/BirthdayProble.lean
+    mathlib archive: https://github.com/leanprover-community/mathlib4/blob/master/Archive/Wiedijk100Theorems/BirthdayProblem.lean
```

Comment:
Ideally we would remove all these `mathlib archive` entries and switch to using `decls`, now that the archive is in doc-gen. That would catch all the typos, and allow linking to the final result.

Reply:
I can do this later today, if desired